platform/autoupdate.cli/src/org/netbeans/modules/autoupdate/cli/ModuleOptions.java

```diff
@@ -175,17 +175,13 @@ protected void process(Env env, Map<Option, String[]> optionValues) throws Comma
                 }
 
                 if (optionValues.containsKey(disable)) {
-                    changeModuleState(optionValues.get(disable), false);
+                    changeModuleState(env, optionValues.get(disable), false);
                 }
 
                 if (optionValues.containsKey(enable)) {
-                    changeModuleState(optionValues.get(enable), true);
+                    changeModuleState(env, optionValues.get(enable), true);
                 }
-            } catch (InterruptedException ex) {
-                throw initCause(new CommandException(4), ex);
-            } catch (IOException ex) {
-                throw initCause(new CommandException(4), ex);
-            } catch (OperationException ex) {
+            } catch (InterruptedException | IOException | OperationException ex) {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**: InterruptedException is ignored. This can delay thread shutdown and clear the thread’s interrupt status. Only code that implements a thread’s interruption policy can swallow an interruption request.

**Fix**: Rethrow the InterruptedException or reinterrupt the current thread using *Thread.currentThread().interrupt()* so that higher-level interrupt handlers can function correctly.
 If you are wrapping the InterruptedException inside a RuntimeException, call *Thread.currentThread().interrupt()* before throwing the RuntimeException.

Learn more about [interrupts](https://docs.oracle.com/javase/tutorial/essential/concurrency/interrupt.html) and [dealing with InterruptedException](https://www.ibm.com/developerworks/java/library/j-jtp05236/index.html?ca=drs-#N10187)